clang/lib/AST/Interp/EvaluationResult.cpp

```diff
@@ -122,19 +124,18 @@ static bool CheckFieldsInitialized(InterpState &S, SourceLocation Loc,
   }
 
   // Check Fields in all bases
-  for (const Record::Base &B : R->bases()) {
+  for (auto [I, B] : llvm::enumerate(R->bases())) {
```

Comment:
Does `auto &[I, B]` work here? What about `const auto &`?

Reply:
I believe `llvm::enumerate` acts like `std::views::enumerate` and iterates by reference, so you’re supposed to just do `auto [...]`; at least that’s what I recall from the documentation.